test/identifier.test.js

```diff
@@ -4,16 +4,16 @@
 const should = require("should");
 const path = require("path");
 
-const identifierUtil = require("../../lib/util/identifier");
+const identifierUtil = require("../lib/util/identifier");
 
-describe("pathUtil", () => {
-	describe("makeRelative", () => {
+describe("util/identifier", () => {
+	describe("makePathsRelative", () => {
 		describe("given a context and a pathConstruct", () => {
 			let context, pathConstruct, expected;
 			beforeEach(() => {
 				context = "/some/dir/";
 				pathConstruct = "/some/dir/to/somwhere|some/other/dir!../more/dir";
-				expected = `${path.relative(context, "/some/dir/to/somwhere")}|${path.relative(context, "some/other/dir")}!${path.relative(context, "../more/dir")}`;
+				expected = `${path.relative("/some/dir/", "/some/dir/to/somwhere")}|some/other/dir!../more/dir`;
```

Comment:
on windows `path.relative("/some/dir/", "/some/dir/to/somwhere")` becomes  `to\somwhere`.
thats not expected, is it? 
As this should yield the same identifier independent of OS?